conversiontracerbullet/src/test/java/org/stanford/AuthDBPropertiesTest.java

```diff
@@ -28,23 +29,24 @@
 
     private AuthDBProperties authProps;
     private Properties serverConf;
-    private String propertyFile;
+    private String serverConfResourceName = "/server.conf";
     private File serverConfFile;
     private File tmpDir;
 
     @Before
     public void setUp() throws IOException, InvocationTargetException, IllegalAccessException, NoSuchMethodException, NoSuchFieldException {
+        tmpDir = createTempDirectory("AuthDBPropertiesTest_").toFile();
         authProps = new AuthDBProperties();
-        // Use private method to get the server.conf file path
-        Field f = AuthDBProperties.class.getDeclaredField("propertyFile");
-        f.setAccessible(true);
-        propertyFile = (String) f.get(authProps);
-        serverConfFile = new File(propertyFile);
-        // Use private method to get the server.conf properties
-        Method m = AuthDBProperties.class.getDeclaredMethod("loadDataSourceProperties", String.class);
+
+        Method m = AuthDBProperties.class.getDeclaredMethod("loadPropertyResource");
```

Comment:
Yet another way to do this is to `assertNotNull`, but that's less specific (although less fragile and does not depend on checking specific values).